README.md

```diff
@@ -140,9 +140,47 @@ dependencies:
 You may also wish to specify constraints on the version, depending on the
 approach to version management that your software otherwise employs.
 
-In order to generate code, you will need to write a tiny Dart program which
-imports the actual builder. We'll assume that you store this program as
-`tool/builder.dart` as seen from the root of your package. Here's the code:
+In order to generate code you will need to run the following command from the
+root directory of your package:
+
+```shell
```

Comment:
FYI, using `console` instead of `shell` (here and below) should give you slightly better syntax highlighting.

Reply:
Thanks!